data/hai/hai.txt

```diff
@@ -207,6 +213,116 @@ phrase "hai"
 		"Lur"
 		"Sen"
 		"Bin"
+		"Bah"
+		"Ki"
+		"Blep"
+		"Tee"
+		"Say"
+		"Hiya"
+		"Choo"
+		"Gip"
+		"Oo"
+		"Mer"
+		"Ki"
+		"Tou"
+
+phrase "hai first names"
+	word
+		"Tee"
+		"Say"
+		"Hiya"
+		"Choo"
+		"Gip"
+		"Oo"
+		"Mer"
+		"Ki"
+		"Tou"
+		"Cor"
+		"Yo"
+		"Sook"
+		"Woa"
+		"Loh"
+		"Eh"
+		"Ow"
+		"Hey"
+		"Lur"
+		"Sen"
+		"Bin"
+		"Na"
+		"Ya"
+		"Os"
+		"Gem"
+		"Mis"
+		"Ey"
+		"Ra"
+		"Des"
+	word
+		"neep"
+		"ari"
+		"mah"
+		"t'k"
+		"eep"
+		"nem"
+		"oot"
+		"ru"
+		"har"
+		"na"
+		"shi"
+		"en"
+		"ana"
+		"ama"
+		"heep"
+		"kei"
+		"yo"
+	word
+		""
```

Comment:
You dont need to add this multiple times if you want to make it more likely, you can put likeliness on these (% of appearance)

Reply:
The more you know! :o